packages/enzyme-matchers/src/assertions/toHaveStyle.js

```diff
@@ -13,14 +13,32 @@ import stringify from '../utils/stringify';
 import html from '../utils/html';
 import single from '../utils/single';
 
+function flattenStyle(style: ?any): ?Object {
```

Comment:
just moved this function above `toHaveStyle` so it is not used before it is defined